modules/improvedigitalBidAdapter.js

```diff
@@ -0,0 +1,415 @@
+var LIB_VERSION_GLOBAL = "3.0.0";
+
+var CONSTANTS = require('src/constants');
+var utils = require('src/utils');
+var bidfactory = require('src/bidfactory');
+var bidmanager = require('src/bidmanager');
+var adloader = require('src/adloader');
+var Adapter = require('src/adapter');
+var adaptermanager = require('src/adaptermanager');
+
+var ImproveDigitalAdapter = function ImproveDigitalAdapter() {
+  var IMPROVE_DIGITAL_BIDDER_CODE = "improvedigital";
+  var baseAdapter = Adapter.createNew(IMPROVE_DIGITAL_BIDDER_CODE);
```

Comment:
`Adapter.createNew` no longer exists.  this should be `var baseAdapter = new Adapter(IMPROVE_DIGITAL_BIDDER_CODE);`